### PR TITLE
loggy loo for node type on startup

### LINKED ISSUE #600: Should runner and API nodes start with different output?
At this moment there's no difference in what fn server writes to logs, would that be runner node or API node.
```
fn-api_1      | time="2017-12-14T17:04:25Z" level=info msg="Serving Functions API on address `:8080`"
fn-api_1      | 
fn-api_1      |         ______
fn-api_1      |        / ____/___
fn-api_1      |       / /_  / __ \
fn-api_1      |      / __/ / / / /
fn-api_1      |     /_/   /_/ /_/
fn-api_1      |         v0.3.235
fn-api_1      | 

```

```
fn-runner_1   | 
fn-runner_1   |         ______
fn-runner_1   |        / ____/___
fn-runner_1   |       / /_  / __ \
fn-runner_1   |      / __/ / / / /
fn-runner_1   |     /_/   /_/ /_/
fn-runner_1   |         v0.3.235
fn-runner_1   | 
fn-runner_1   | time="2017-12-14T17:04:27Z" level=info msg="Serving Functions API on address `:8080`"

```
So, should fn "instance" log its node type?

### PR DESCRIPTION
**Pull request by rdallman**
closes #600

### CODE REVIEW

File: api/server/server.go [hunk 1]
```diff
@@ -319,7 +330,7 @@ func (s *Server) startGears(ctx context.Context, cancel context.CancelFunc) {
 	fmt.Println(runHeader)
 	fmt.Printf("        v%s\n\n", version.Version)
 
-	logrus.Infof("Serving Functions API on address `%s`", listen)
+	logrus.WithField("type", s.nodeType).Infof("Serving Functions API on address `%s`", listen)
```
Comment (denismakogon) [inline api/server/server.go]: Cosmetic thing:
```go
logrus.Infof("Serving Functions %v API on address `%v`", s.nodeType, listen)
```
this is what i've been thinking about.

Comment (rdallman) [reply api/server/server.go]: makes sense too. 1s

**Comment by rdallman** [comment]
@denismakogon updated

File: api/server/server.go (hunk 1 above)
Comment (treeder) [reply api/server/server.go]: +1 to @denismakogon suggestion, except drop API and add %s in place so it won't be `api API`. 

Orrrr, while we're nitting this, how about change it to this:

```go
logrus.WithField("type", s.nodeType).Infof("Fn serving on `%s`", listen)
```

Comment (rdallman) [reply api/server/server.go]: thank you sir may I please have another

**Comment by rdallman** [comment]
@treeder updated

**Pull request merged**